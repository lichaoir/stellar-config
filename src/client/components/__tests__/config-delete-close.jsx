const React = require('react');
const renderer = require('react-test-renderer');
const Close = require('../config-delete-close.jsx');

describe('component config-delete-close', () => {
  test('is clickable', () => {
    const props = {
      handleClick: jest.fn(),
    };

    const component = renderer.create(<Close {...props} />);
    const tree = component.toJSON();
    tree.props.onClick();

    expect(tree).toMatchSnapshot();
    expect(props.handleClick).toHaveBeenCalledTimes(1);
  });
});
