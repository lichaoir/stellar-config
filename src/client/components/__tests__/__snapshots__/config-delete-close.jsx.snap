// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component config-delete-close is clickable 1`] = `
<button
  className="delete"
  onClick={[Function]}
/>
`;
